src/library_nodefs.js

```diff
@@ -1,5 +1,5 @@
 mergeInto(LibraryManager.library, {
-  $NODEFS__deps: ['$FS', '$PATH'],
+  $NODEFS__deps: ['$PATH'],
```

Comment:
It's ok to have circular deps, but only having circular deps with postsets is the problem, correct? Then perhaps it could be less confusing to remove the postsets from NODEFS etc., and move them to FS (and keep the circular deps).